client/extensions/woocommerce/app/settings/shipping/shipping-labels.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormFieldSet from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import FormToggle from 'components/forms/form-toggle';
+import ShippingCard from './shipping-card';
+import ShippingHeader from './shipping-header';
+
+class ShippingLabels extends Component {
+	constructor( props ) {
+		super( props );
+
+		//TODO: use redux state with real data
+		this.state = {
+			visible: true,
+			cards: [ {
+				selected: true,
+				type: 'VISA',
+				digits: '1234',
+				name: 'Name Surname',
+				date: '12/19'
+			}, {
+				selected: false,
+				type: 'MasterCard',
+				digits: '5678',
+				name: 'Name Surname',
+				date: '01/20'
+			} ]
+		};
+
+		this.onToggle = this.onToggle.bind( this );
```

Comment:
We can remove these binds and instead use arrow functions, which is the direction Calypso is heading in. It saves updating these in two places in the future. https://babeljs.io/blog/2015/06/07/react-on-es6-plus#arrow-functions. We discussed this a bit in calypso-framework earlier. So you can drop this line, and then do `onToggle = () => {` below. Ditto with `renderCard`.